framework/SwiftPlot/Series.swift

```diff
@@ -1,13 +1,21 @@
 // struct defining a data series
 public struct Series {
 
+	public enum hatching: Int, CaseIterable{
+		case none = 0
+		case forwardSlash = 1
+		case backwardSlash = 2
+	}
+	public var hatchPattern: hatching = .none
```

Comment:
could things that are plot-specific (like hatchPattern is specific to bar chart) go into some plot-specific "series options" datastructure?

e.g. `BarChartSeriesOptions { public var hatchPattern: hatching }`

Reply:
I'll make the change